src/app/SharpRaven/Data/SentryStacktrace.cs

```diff
@@ -91,7 +84,7 @@ public override string ToString()
 
             StringBuilder sb = new StringBuilder();
 
-            foreach (var frame in Frames)
+            foreach (var frame in Frames.Reverse())
```

Comment:
How about asserting that the string contains the type names of the exceptions, but no assertions of the overall format of the string?